chart/values.yaml

```diff
@@ -3,11 +3,15 @@
 # Declare variables to be passed into your templates.
 
 mayastorCP:
-  registry:
+  registry: ''
+  tag: latest
   pullPolicy: Always
   logLevel: info
 
 base:
+  cache_poll_period: ''
```

Comment:
So these are actually read from [here](https://github.com/openebs/mayastor-control-plane/blob/develop/chart/constants.yaml) when using the script. By I guess it's fine to leave it as empty by default.

Reply:
Yes, I saw that `constants.yaml` gets generated, but `helm` doesn't know about that file. I think leaving it empty will use the same defaults since it will just reused the defaults from the executables themselves.